### DISCUSSION TITLE
Dashboard Widgets

### DISCUSSION THREAD

**Discussion by dev-dhull** [filamentphp/filament#3177] [Help] [answered]
How to remove the logo and documentation URL of filament on the filament dashboard?

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/admin/dashboard/getting-started#disabling-the-default-widgets

**Reply by ghost** [reply]
If you're using FilamentPHP version 3, follow these steps:

Go to the panel from which you want to remove the default widgets.

**Update the below code**

```php
->widgets([
    Widgets\AccountWidget::class,
    Widgets\FilamentInfoWidget::class,
])
```

**To this**

```php
->widgets([])
```

Official documentation:
https://filamentphp.com/docs/3.x/panels/dashboard#disabling-the-default-widgets

**Comment by mrleeuk** [comment]
To remove the logo, set 

'footer' => [
            'should_show_logo' => false,

in config.

**Reply by malikwaqasmustafa** [reply]
no solution worked for me Tried putting it in the filament.php.

I ended up fixing it by commenting the two default widgets in 'app/Providers/Filament/AdminPanelProvider.php' 

find 

`->widgets([
                Widgets\AccountWidget::class,
                Widgets\FilamentInfoWidget::class,
            ])` 
comment these two items of array and it will work definitely.
